mmv1/products/notebooks/terraform.yaml

```diff
@@ -116,6 +116,53 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         ignore_read: true
       shieldedInstanceConfig: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+  Runtime: !ruby/object:Overrides::Terraform::ResourceOverride
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 15
+      update_minutes: 15
+      delete_minutes: 15
+    autogen_async: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "notebook_runtime_basic"
+        primary_resource_id: "runtime"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-notebooks-runtime%s\", context[\"random_suffix\"])"
+        region_override: "us-central1"
+        vars:
+          runtime_name: "notebooks-runtime"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "notebook_runtime_basic_gpu"
+        primary_resource_id: "runtime_gpu"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-notebooks-runtime-gpu%s\", context[\"random_suffix\"])"
+        region_override: "us-central1"
+        vars:
+          runtime_name: "notebooks-runtime-gpu"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "notebook_runtime_basic_container"
+        primary_resource_id: "runtime_container"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-notebooks-runtime-container%s\", context[\"random_suffix\"])"
+        region_override: "us-central1"
+        vars:
+          runtime_name: "notebooks-runtime-container"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "notebook_runtime_full"
+        primary_resource_id: "runtime_full"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-notebooks-runtime-full%s\", context[\"random_suffix\"])"
+        region_override: "us-central1"
+        vars:
+          runtime_name: "notebooks-runtime-full"
+    description: |
+      {{description}}
+
+      ~> **Note:** Due to limitations of the Notebooks Instance API, many fields
+      in this resource do not properly detect drift. These fields will also not
+      appear in state once imported.
+    # `ignore_read` prevents update ie destroy when testing.
+    properties:
+      softwareConfig: !ruby/object:Overrides::Terraform::PropertyOverride
+        ignore_read: true
+      virtualMachine: !ruby/object:Overrides::Terraform::PropertyOverride
+        ignore_read: true
```

Comment:
this seems like a pretty big deal and we should *not* do this

If we ignore_read on these top-level fields terraform won't notice a diff and will not realize if something has been changed remotely and needs to be reconciled with the config

Reply:
Indeed, completely slipped my mind. Thanks for the feedback. I removed the `ignore_read` and addressed the fields that caused tests to fail.